webui/react/src/components/kit/NoteCard.tsx

```diff
@@ -6,31 +6,29 @@ import { unstable_useBlocker as useBlocker } from 'react-router-dom';
 import Button from 'components/kit/Button';
 import Input from 'components/kit/Input';
 import Tooltip from 'components/kit/Tooltip';
+import Markdown from 'components/Markdown';
 import Spinner from 'shared/components/Spinner/Spinner';
 import { ErrorType } from 'shared/utils/error';
 import handleError from 'utils/error';
 
-import Markdown from './Markdown';
-import css from './NotesCard.module.scss';
+import css from './NoteCard.module.scss';
```

Comment:
At least some of this CSS can probably be eliminated by using other UI Kit components. Can you take a stab at seeing what's removable?